operating/manage-user-access/manage-role-based-access-control.adoc

```diff
@@ -0,0 +1,33 @@
+[id="manage-role-based-access-control"]
+= Managing role-based access control
+include::modules/common-attributes.adoc[]
+:context: manage-role-based-access-control
+
+toc::[]
+
+[role="_abstract"]
+{product-title} comes with Role-based access control (RBAC) that you can use to configure roles and grant various levels of access to {product-title} to different users.
+These roles govern access to {product-title} resources to prevent unwanted access.
+
+* *Roles* are a collection of rules, which are a group of `read` and `write` permissions that user can perform on a set of resources.
```

Comment:
We usually use italics instead of bold when introducing a term like "roles" and "resources" here